src/roi/Roi.ts

```diff
@@ -387,6 +395,8 @@ export class Roi {
 
   /**
    * Returns the diameter of a circle of equal projection area
```

Comment:
Prefer "Computes" over "Returns". The fact that it "returns" is already documented.